pyron-core/src/main/scala/com/cloudentity/pyron/domain/flow/flow.scala

```diff
@@ -45,15 +45,14 @@ object PathMatching {
     pathPrefix.value + pathPattern.value
 
   def createPatternRegex(rawPattern: String): Regex = {
-    val regex = namePlaceholderRegex.findAllIn(rawPattern).toList.foldLeft(rawPattern) { case (acc, mtch) =>
-      val name = mtch.drop(1).dropRight(1)
-      acc.replaceFirst(namePlaceholderPattern, s"(?<$name>[^/]+)")
-    }
+    val regex = namePlaceholderRegex.replaceAllIn(rawPattern, m => s"(?<${m.group(1)}>[^/]+)")
```

Comment:
As usual with regex, this is criptic. Could we have a comment describing what this does? Or maybe a descriptive variable name ?

Reply:
ok, will do